.github/workflows/ci.yaml

```diff
@@ -15,25 +15,26 @@ jobs:
       fail-fast: false
       matrix:
         # different operating systems
-        os: [ubuntu-18.04, macos-10.15, windows-2019]
-        features: [--features default]
+        os: [ubuntu-18.04, ubuntu-20.04, macos-10.15, windows-2019]
+        features: [default]
         toolchain: [stable]
         include:
           # different features combinations
           - os: ubuntu-18.04
-            features: --features beta
+            features: beta
           - os: ubuntu-18.04
-            features: --features blocking
+            features: blocking
           - os: ubuntu-18.04
-            features: --features "blocking beta"
+            features: "blocking beta"
           # different tls implementations
           - os: ubuntu-18.04
-            features: --features tls-rustls --no-default-features
+            features: tls-rustls --no-default-features
           - os: ubuntu-18.04
-            features: --features tls-vendored-openssl --no-default-features
+            features: tls-vendored-openssl --no-default-features
           # MSRV
           - os: ubuntu-18.04
             toolchain: 1.41.1
+            features: default
```

Comment:
this is redundant with `features: [default]` above, no?

Reply:
No, the above is a matrix that just changes the OS and has set features/toolchain. Down here are individual combinations added to the matrix, so they don't inherit the `features: [default]` from above. So without this line, the `cargo test` line gets made because it has `--features` with nothing specified.